server/routes/createThread.ts

```diff
@@ -21,8 +23,86 @@ export const Errors = {
   InsufficientTokenBalance: `Users need to hold some of the community's tokens to post`,
 };
 
+// TODO - perhaps find a way to make this and client/scripts/controllers/app/token.ts 
```

Comment:
You could expose the results here as a route for the client to fetch.

Reply:
moreover, you could create a cache on our server for the token lists for faster load times

Reply:
Great ideas